docs/pages/API/animals.mdx

```diff
@@ -0,0 +1,236 @@
+# Animal
+
+Module to generate animal related data.
+
+## Overview
+
+For a general type of animal (e.g. `'dog'`), use [animal()](#animal).
+
+Otherwise, use one of the more specific methods, such as cat() for a specific breed of cat.
+
+All values may be localized.
+
+## bear
+Returns a random bear species.
+
+***Returns***: string
+
+```rust
+use data_faking::locales::en as faking_en;
+
```

Comment:
`use data_faking::locales::en as faking_en;` I think it's better to change this simply to `faking` so if someone wants to change locales in the future from the imports they don't need to change all the function calls as well to match `faking_{lang_code}`.